### PR TITLE
Use relative links

### PR DESCRIPTION
**Pull request by Baschdl**
- Exchanged absolute links by relative links
- Added more descriptive alt text

### CODE REVIEW

File: people/sebastian-bischoff.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ author_profile: true
 ## Sebastian Bischoff
 
 
-<img src="https://github.com/claassenlab/claassenlab.github.io/blob/master/people/sebastian-bischoff.jpg?raw=true" alt="Sebastian bischoff" width="144" height="144">
-
+<img src="../sebastian-bischoff.jpg" alt="Photo of Sebastian Bischoff" width="144" height="144">
+<!-- ![alt text](../sebastian-bischoff.jpg#144x144 "Photo of Sebastian Bischoff") -->
```
Comment (Baschdl) [inline people/sebastian-bischoff.md:14]: @V-Sehra Did you find an option to resize images with markdown? Adding the image this way works, resizing it doesn't

Comment (V-Sehra) [reply people/sebastian-bischoff.md:14]: tested the lines did not work sadly

Comment (Baschdl) [reply people/sebastian-bischoff.md:14]: You mean line 14 doesn't work? Line 13 does work

Comment (Baschdl) [reply people/sebastian-bischoff.md:14]: image

**Comment by Baschdl** [comment]
@V-Sehra Does anything prevent us from merging this?

**Pull request merged**